SR2024/2024-03-15-competition-details.md

```diff
@@ -0,0 +1,64 @@
+---
+to: Student Robotics 2023
+subject: SR2024 Competition Details
+---
+
+Hello everyone!
+
+It's just under a month until the big day - [the competition][competition-event-page]!
+
+Before then, we have a few important updates and reminders for you:
+
+## Attendance details
+
+Thanks to everyone who has already provided their attendance details. If you haven't done that yet, or things have changed substantially please [provide attendance details](https://forms.gle/8YhfdRbfDWP3KKNy8).
+
+Doors open on both days at 9am and each day will finish around 5:30pm. The event will formally start at 10:30am on Saturday. There are now full day schedules on the [event page][competition-event-page]. **If you are expecting to arrive after 10am on Saturday, please let us know.**
+
+You'll have access to your pits and the test arena when you arrive, giving your team time to work on their robot before things get going in earnest.
+
+You're welcome to bring family and friends as guests, however please note that space in the pits is limited.
+
+We'll be publishing rules for behaviour in/around the competition venue when you arrive. If you have any questions or concerns beforehand, please let us know.
+
+The competition this year is a public event, and we do not have exclusive access to the venue.
```

Comment:
TODO: verify this is accurate.

See https://studentrobotics.slack.com/archives/CBP7UL6RG/p1710517484320509?thread_ts=1710517316.696579&cid=CBP7UL6RG.

Reply:
f3f1051ca7b86d0824bae8cc394025ba9db24571